flutter-poet.py

Rayredignton opened the pull request:
[flutter-poet.txt](https://github.com/jgodstime/team-fury-1/files/4734499/flutter-poet.txt)

Crestward committed:
updated python file

obiwanpelosi committed:
update obiwan.js

Commit:
Add js script

epospiky committed:
added my script

dave-ok committed:
Merge pull request #101 from focus2000/dev

kenneth  Nwachukwu script

thebolarin committed:
Merge pull request #100 from MKaneto/dev

Kaneto script.js

qobitech committed:
added sticklo.js to scripts folder

Commit:
add my script

qobitech committed:
Merge pull request #414 from sticklo/sticklo

added sticklo.js to scripts folder

Damilare50 committed:
My updated script

dnor-dev committed:
Merge branch 'dev' into DNor

onyxdelagee committed:
onyxdelagee-HNG-00136 updated script review changes

Damilare50 committed:
Update Damilare2020.js

marknesh committed:
merge pull request

thebolarin committed:
Merge pull request #98 from dan-ibeh/daniel-hng

Daniel hng script

Sambalicious committed:
script file for sambalicious

thebolarin committed:
Merge pull request #419 from onyxdelagee/HNG-00136

onyxdelagee-HNG-00136 updated script review changes

qobitech committed:
Merge pull request #415 from forte001/samson-dev

Final fixes

thebolarin committed:
Merge pull request #97 from TonyNgeno/addedfile

added file

z3k9 committed:
add my corrected script

qobitech committed:
Merge pull request #420 from marknesh/dev

merge pull request

sheuabdulazeez committed:
Added my script

qobitech committed:
Merge branch 'dev' into Damilare2020_branch1

qobitech committed:
Merge pull request #418 from Damilare50/Damilare2020_branch1

Damilare2020 branch1

qobitech committed:
Merge pull request #417 from asapkenny3/arohakeem

add my script

sheuabdulazeez committed:
Delete ayomadewale.php

qobitech committed:
Merge pull request #416 from dnor-dev/DNor

DNor's script added successfully

qobitech committed:
Merge pull request #413 from RogueCode007/myBranch

update obiwan.js

Damilare50 committed:
Update Damilare2020.js

FrankKamala committed:
submit My Script

qobitech committed:
Merge pull request #423 from Z3k9/diffBranch

add my corrected script

qobitech committed:
Merge pull request #422 from Sambalicious/dev

script file for sambalicious

sheuabdulazeez committed:
added my script

FrankKamala committed:
comment on my Script

FaithMo committed:
Merge pull request #372 from mocktarissa/patch-3

Added email in the script

qobitech committed:
Merge pull request #412 from Crestward/dev

Updated python script

Commit:
DamiCode script written in JavaScript

qobitech committed:
Merge pull request #411 from costamay/Titus

Add js script

qobitech committed:
Merge pull request #410 from epospiky/dev

added my script

thebolarin committed:
Merge pull request #386 from TomiTsuma/dev

Please merge this

qobitech committed:
Merge pull request #427 from jgodstime/add-file

ekiminfo2.js

qobitech committed:
Merge pull request #426 from Tijatech/dev

My Task 1 script

qobitech committed:
Merge pull request #425 from Damilare50/Damilare2020_branch1

Update Damilare2020.js

marknesh committed:
push for merge

qobitech committed:
Merge pull request #424 from Menor25/menor

My HNG username is Menor, this is my commit

thebolarin committed:
Merge pull request #388 from Idss15/Subomi

Subomi Pull Request

onyxdelagee committed:
onyxdelagee-HNG-00136 script review final

qobitech committed:
Merge branch 'dev' into Zinny

qobitech committed:
Merge pull request #409 from Zinniee/Zinny

add my script

qobitech committed:
Merge pull request #408 from BenjaminKyamanywa/benjamin-hng-submission

Benjamin Kyamanywa HNG stage 1 submission

thebolarin committed:
Merge pull request #390 from samliks/patch-1

Update samliks.js

qobitech committed:
Merge pull request #407 from RogueCode007/newBranch

add obiwan.js

thebolarin committed:
Merge pull request #391 from Ekaji/patch-1

Update ekaji.js

AyoPips committed:
Update AyoPips.js

Deenladan committed:
Create Deenladan.js

qobitech committed:
Merge pull request #80 from alexismajya/alexisScript-00521

added-my-script

qobitech committed:
Merge pull request #78 from alxzndr2019/dev

submitted HNG task

qobitech committed:
Merge pull request #74 from DerKaiser7/HNG-02844

added derkaiser HNG-02844 script

Deenladan committed:
Merge pull request #1 from Deenladan/Deenladan-script

Create Deenladan.js

qobitech committed:
Merge pull request #69 from mustafa-aisha/maisha-hng-task2

this is my commit

FaithMo committed:
Merge pull request #294 from 69codes/branches

add this thing now, and it must work

thebolarin committed:
Merge pull request #396 from mechetenco/dev

add my script

marknesh committed:
push for merge

FaithMo committed:
Merge pull request #297 from privatesaint/hng-hw

Stage one task done

AyoPips committed:
Merge branch 'dev' into AyoPIps-new-branch

qobitech committed:
Merge pull request #434 from AyoPips/AyoPIps-new-branch

Update AyoPips.js

FaithMo committed:
Merge pull request #306 from JoanneLopez/HNG-00618

Joannelopez-HNG-00618-08067870176

qobitech committed:
Merge pull request #431 from onyxdelagee/HNG-00136

onyxdelagee-HNG-00136 script review final

thebolarin committed:
Merge pull request #385 from djsmacker01/dev

Task complete

qobitech committed:
Merge pull request #430 from DamiCode18/DamiCode

DamiCode script written in JavaScript

qobitech committed:
Merge pull request #428 from FrankKamala/kamala

Kamala

thebolarin committed:
Merge pull request #398 from Pholarin/dev

Task2

qobitech committed:
Merge pull request #403 from abiiagarin/abigailagarin

add my script for task

qobitech committed:
Merge pull request #401 from ekiminfo/patch-2

ekiminfo.js

qobitech committed:
Merge pull request #384 from NwaiwuVictoria/NwaiwuVictoria

add my script

qobitech committed:
Merge pull request #383 from Iamprashanth-1/dev

Create nenu.py

qobitech committed:
Merge pull request #355 from TonyOuma/dev

fix declare variables globally

dev-bazz committed:
myscript

qobitech committed:
Merge pull request #356 from Ernest-Opara/correction

Corrected the error supposedly causing it to fail

qobitech committed:
Merge pull request #353 from jerrynoah96/Jeremiah2407

Jeremiah2407 added Js file

Mrkellard committed:
Delete mrkellard

FaithMo committed:
Merge pull request #317 from fkelonthebeat/fkelonthebeat-HNG-06361

Adding fkelonthebeat's script for stage 2

qobitech committed:
Merge pull request #346 from olaoluwa057/FatoluOlaoluwa

 olaoluwa added olaoluwa.js

qobitech committed:
Merge pull request #345 from fir3fly89/ORscript

ORscript

qobitech committed:
Merge pull request #343 from adun-dotcom/Odetola

add my script for HNGi7  stage 2 task

thebolarin committed:
Merge pull request #436 from BaZpr/dev

BaZz

FaithMo committed:
Merge pull request #301 from pauxiel/dev

added my script

dave-ok committed:
Merge pull request #56 from busgandy/master

Add files via upload

FaithMo committed:
Merge pull request #318 from AbisoyeAlli/patch-1

Updated my script file

FaithMo committed:
Merge pull request #319 from scotlopez/scotlopez

Javascript task with name and Hngid by Scotlopez

forte001 committed:
Update Samson_Babalola.py

qobitech committed:
Merge pull request #333 from devGeorgeOwi/newDevBranch

added my script georgeowi

bvadegboye committed:
PLEASE MERGE MY NEW COMMIT

qobitech committed:
Merge pull request #337 from PillarBassey/pillarBassey-HNG-01529

pillarBassey-HNG-01529 script

qobitech committed:
Merge pull request #339 from oelun-in/olajide

Olajide oluniyi commited his script

thebolarin committed:
Merge pull request #404 from adekniyi/patch-1

Update AdekNiyi.js

FaithMo committed:
Merge pull request #321 from varlov/dev

Please, review

thebolarin reviewed (changes requested):
Please add only your script file

dave-ok committed:
Merge pull request #89 from Christie-code/HNGbranch

Added my script file

thebolarin committed:
Merge pull request #377 from Marvvy-G/marvvy-dev

Marvvy Stage 2 Task

onyxdelagee committed:
onyxdelagee-HNG-00136 script review final2

dave-ok committed:
Merge pull request #94 from Chinwenmeri/dev

chukwudi's script commit

thebolarin committed:
Merge pull request #435 from marknesh/dev

push for merge

dave-ok committed:
Merge pull request #96 from AyoAyomide/ayomide

ayoayomide task 1

thebolarin committed:
Merge pull request #438 from onyxdelagee/HNG-00136

onyxdelagee-HNG-00136 script review final2

thebolarin committed:
Merge pull request #437 from forte001/samson-dev

Updated Samson_Babalola.py

nyior committed:
modified variale names

qobitech committed:
Merge pull request #272 from bvadegboye/VictoriaBranch

Create Victoria.js

elsie23-hash committed:
Create Elsie_Okoli.py

frank9th committed:
added Frank Moses to scripts

Arch997 committed:
Add my script

tiVu99 committed:
Add script

MuchesiaKe committed:
add my script

Commit:
Remove the whitespace on script

Commit:
I added my js file which is AyoMicho.js

nyior committed:
added semi - colons. don't want to take chances

FunsoAdetola committed:
Add files via upload

nyior committed:
added email to output string

Damilare50 committed:
Update Damilare2020.js

AbisoyeAlli committed:
Update Abisoye.js

elsie23-hash committed:
Elsie_Okoli.py

script for stage 1

alexismajya committed:
add-email-to-script

lavender2000 committed:
create lasunkanmi.js

please merge and run my script

dave-ok committed:
Merge pull request #202 from abu4kind/dev

Abu4kind_my hng7i task for team fury-1

GideonMoses11 committed:
gedoni script added

dave-ok committed:
Merge pull request #225 from ekpenyongmike/project

new file added davemike

BoovyBernard committed:
Update kizito_bernard.py

Smizzy01 committed:
Delete yyy.py

dave-ok committed:
Merge pull request #329 from Nyior/updated_script

fixed minor issues in my script

dave-ok committed:
Merge pull request #174 from Smizzy01/Smizzy

Smizzy.py

qobitech committed:
Merge pull request #453 from MuchesiaKe/MuchesiaBranch

Wallace Muchesia

qobitech committed:
Merge pull request #459 from lavender2000/patch-1

create lasunkanmi.js

dev-bazz committed:
my latest update to script

qobitech committed:
Merge pull request #368 from Fortunate245/fortunate

Add Fortunate to Scripts

Marvvy-G committed:
Create correction.js

qobitech committed:
Merge pull request #444 from frank9th/dev

 added Frank Moses to scripts

qobitech committed:
Merge pull request #456 from Arch997/archibong

Add my script

Ajaodolapo1301 committed:
last

qobitech committed:
Merge pull request #283 from Elelu/patch-1

Create moh_elelu.py

elsie23-hash committed:
Delete Elsie_Okoli.py

delight500 committed:
added my script

dev-bazz committed:
my latest update

dave-ok committed:
Merge pull request #253 from mathjken/HNG-04902

Ogu Johnkennedy

qobitech committed:
Merge pull request #457 from elsie23-hash/dev

my pull request

kaycee97 committed:
new script

qobitech committed:
Merge pull request #375 from francisudeji/francisudeji-update

Fix typo

qobitech committed:
Merge pull request #472 from kaycce/HNG00363

Hng00363

qobitech committed:
Merge pull request #469 from Ajaodolapo1301/last

last

thebolarin committed:
Merge pull request #470 from BaZpr/dev

BaZz

thebolarin committed:
Merge pull request #471 from delight500/Delight

added my script

thebolarin committed:
Merge pull request #468 from Marvvy-G/new-branch-marv

Marv correction script

dev-bazz committed:
Update BaZz.js

thebolarin committed:
Merge pull request #464 from BoovyBernard/dev

Update 2 kizito_bernard.py

thebolarin committed:
Merge pull request #473 from BaZpr/dev

Update BaZz.js

thebolarin committed:
Merge pull request #458 from Damilare50/Damilare2020_branch1

Update Damilare2020.js

Sorin-tech committed:
add my script

z3k9 committed:
corrected errors

thebolarin committed:
Merge pull request #460 from GideonMoses11/gedonihng

gedoni script added

thebolarin committed:
Merge pull request #461 from AyoMicho/AyoMicho

I added my js file which is AyoMicho.js

thebolarin committed:
Merge pull request #446 from Deenladan/dev

deenladan.js

thebolarin committed:
Merge pull request #445 from tiVu99/dev

Add script

dave-ok committed:
Merge pull request #324 from AbisoyeAlli/patch-2

Abisoye updated script

obiwanpelosi committed:
update obiwan.js

69codes committed:
na God go forgive me for my mistakes

Brownexia committed:
Add my script

dave-ok committed:
Merge pull request #455 from alexismajya/add-email-to-script

add-email-to-script

qobitech committed:
Merge pull request #477 from Sorin-tech/sorinbranch

add my script

Damflex committed:
Improved Damflex.js

Trojan9 committed:
Add my script

gkarumbi committed:
Create object to hold my details

Ajaodolapo1301 committed:
las

Francisop committed:
update francis.py

qobitech committed:
Merge pull request #487 from Ajaodolapo1301/me

las

qobitech committed:
Merge pull request #486 from gkarumbi/gkarumbi

Create object to hold my details

qobitech committed:
Merge pull request #280 from g-prog/g-prog

G prog

dave-ok committed:
fix: remove stragglers modify name regx

qobitech committed:
Merge pull request #485 from Trojan9/team-fury

Add my script

qobitech committed:
Merge pull request #479 from 69codes/branches

na God go forgive me for my mistakes

Thobeats committed:
Update thobeats.php

qobitech committed:
Merge pull request #483 from naomianthony/jahsom-1

please add this corrected script by jahsom

Saviourise committed:
Create Saviee.java

dave-ok committed:
Merge pull request #495 from dave-ok/bg-modify-name-matcher

[hot-fix]:modify name regx

qobitech committed:
Merge pull request #474 from Z3k9/differentBranch

corrected errors, added email

qobitech committed:
Merge pull request #496 from Saviourise/patch-3

Create Saviee.java

qobitech committed:
Merge pull request #482 from Brown-coder/myNewBranch

Add my script

Francisop committed:
Update francis.py

elsie23-hash committed:
Elsie_Okoli.py

qobitech committed:
Merge pull request #494 from Francisop/patch-1

update francis.py

nyior committed:
using var now not const

FunsoAdetola committed:
Delete funsoadetola.py

dave-ok committed:
Merge pull request #327 from EmmyGee/nwachi

kindly add my script - HNGi7 stage 2 task

frank9th committed:
Corrected Hngi7 Id number of Frank Moses

FunsoAdetola committed:
Add files via upload

nyior committed:
fixed spacing

omebu committed:
Add files via upload

stage 2 task

Ekaji committed:
Merge branch 'dev' into patch-2

simply-alliv committed:
docs: update readme with improved instructions

simply-alliv committed:
Update README.md

simply-alliv committed:
Update README.md

obiwanpelosi committed:
update obiwan.js

Marvvy-G committed:
Update marvvy.js

Marvvy-G committed:
Update marvvy.js

Ekaji committed:
Update ekaji.js

dnor-dev committed:
DNor's script added successfully

naomianthony committed:
add my modified script by jahsom

Karrim-real committed:
Update karrim4real.php

Damflex committed:
my script improved

dro-1 committed:
Merge pull request #540 from Karrim-real/patch-2

Update karrim4real.php

piouson committed:
Add gitpod for live review

Resolve #543

dro-1 committed:
Merge pull request #544 from Marvvy-G/marvvy-dev

Marvvy dev

frank9th committed:
Reviewed as requested. 

I was told i didn't print out my output, but i have checked and the code is alright. please review and me more specific with what is need. thanks.

fkelonthebeat committed:
Update fkelonthebeat-HNG-06361.js

omebu committed:
Add files via upload

fkelonthebeat committed:
Update fkelonthebeat-HNG-06361.js

dro-1 committed:
Merge pull request #322 from FunsoAdetola/dev

edited copy

dro-1 committed:
Merge pull request #298 from otunbaoke1/myNewBranch

my pull request

dro-1 committed:
Merge pull request #288 from Cheesom/myNewDev

myData Output in javascript

Mrkellard committed:
Update mrkellard.js

dro-1 committed:
Merge pull request #217 from ryann254/HNG-00306

Ryan_Njoroge-Hng 00306

funshi committed:
Rename funshi HNG-04915 python to Funshi.py

dro-1 committed:
Merge pull request #480 from RogueCode007/myBranch

update obiwan.js

dro-1 committed:
Merge pull request #449 from costamay/Titus

Remove the whitespace on script

dro-1 committed:
Merge pull request #392 from Ekaji/patch-2

Minor Update ekaji.js

dro-1 committed:
Merge pull request #371 from jebitok-dev/hngi7

task script

dro-1 committed:
Merge pull request #370 from ThatSameBlackBoyFromAfrica/dev

HNGi7 team-fury-1 task

hng-teamfury committed:
Merge pull request #546 from piouson/add-gitpod

Add gitpod for live review

piouson committed:
Add apache config file

Resolve #546

dro-1 committed:
Merge pull request #493 from Damflex/dev

Improved Damflex.js

Rubylena committed:
Merge pull request #264 from funshi/patch-1

funshi.py

dro-1 committed:
Merge pull request #497 from Thobeats/thobeats

Update thobeats.php

dro-1 committed:
Merge pull request #498 from elsie23-hash/patch-1

Elsie_Okoli.py

hng-teamfury committed:
Merge pull request #561 from piouson/add-apache

Add apache config file

Rubylena committed:
Merge pull request #308 from Mrkellard/mrkellard

Create mrkellard.js

abu4kind committed:
my hng7i task one for team fury

harriemman committed:
Create solace.js

Rubylena committed:
Merge pull request #524 from simply-alliv/docs

Update README

dro-1 committed:
Merge pull request #506 from frank9th/dev

 Corrected Hngi7 Id number of Frank Moses

dro-1 committed:
Merge pull request #507 from Nyior/updated_script

Updated script

dro-1 committed:
Merge pull request #508 from chigboraphael/dev

Stage 2 task.

dro-1 committed:
Merge pull request #513 from babafemi99/babafemi99

Oloniyo Oreoluwa task 2 submission

simply-alliv committed:
Merge pull request #549 from fkelonthebeat/patch-1

Update fkelonthebeat-HNG-06361.js

dro-1 committed:
Merge pull request #566 from Idss15/Subomi

My Pull Request

dro-1 committed:
Merge pull request #565 from abu4kind/dev

Task completed

dro-1 committed:
Merge pull request #556 from funshi/funshi-patch-2

Olufunsho.py

Rubylena committed:
Merge pull request #323 from EkeChimdi/chimdiebube

Chimdi HNG task script in python

dro-1 committed:
Merge pull request #568 from harriemman/dev

solace.js

Rubylena committed:
Merge pull request #367 from delight500/dev

my script upload

simply-alliv committed:
Merge pull request #400 from naomianthony/jahsom

add my modified script by jahsom

Rubylena commented:
resend the PR the right way.

naomianthony committed:
add this corrected script by jahsom

Idss15 committed:
Add my Python script

PillarBassey committed:
pillarBassey-HNG-01529 script
